bigchaindb/pipelines/block.py

```diff
@@ -166,7 +166,7 @@ def create_pipeline():
     """Create and return the pipeline of operations to be distributed
     on different processes."""
 
-    block = Block()
+    block = BlockPipeline()
 
     block_pipeline = Pipeline([
```

Comment:
Probably swap around the names; `block_pipeline = BlockPipeline()` and `pipeline = Pipeline(...)`.